src/components/summaryCart/UserOrder.tsx

```diff
@@ -0,0 +1,64 @@
+import { PizzaOrder } from '@ts/types/types.js';
+import { v4 as uuidv4 } from 'uuid';
+import { decrementPizza, addPizza, deletePizza } from '@store/pizzaSlice.js';
+import { CURRENCY } from '@constants/constants.js';
+import React from 'react';
+import { PayloadAction } from '@reduxjs/toolkit';
+
+export default function userOrder({
+  order,
+  dispatch,
+}: {
+  order: PizzaOrder[];
+  dispatch: (pizza: PayloadAction<PizzaOrder>) => void;
+}) {
+  return order?.map((el: PizzaOrder) => {
```

Comment:
order всегда должен приходить, даже если он пустой
```suggestion
  return order.map((el: PizzaOrder) => {
```